### PR TITLE
Add override target name option

### PR DESCRIPTION
**Pull request by joannanosek**

### CODE REVIEW

**Comment by dtrawins** [comment]
Currently grpc use 2 functions imported from common folder. It would be easier to reuse the client if all content is in one location. Could you move the files from common folder to example/grpc_client? This way all client related content would be in a single folder. For the file endpoint_utlil.py perhaps a better name would be grpc_client_utils. It will require the tests using grpc client to import all functions from the new folder instead of common.
The rest of the change looks ok to me.

**Comment by joannanosek** [comment]
@dtrawins it cannot be done as e2e tests are using common folder as well

**Comment by dtrawins** [comment]
@joannanosek The tests could be just switched to use the those functions from new folder
Just the import in test_inference_endpoints.py should be changed. This way we could drop the common folder. The tests would rely on the grpc_client.

File: examples/grpc_client/README.md [hunk 1]
```diff
@@ -24,22 +24,72 @@ pip install -r requirements.txt
...
+  --image_size IMAGE_SIZE
+                        Size of images. Default: 224
+  --images_number IMAGES_NUMBER
+                        Define number of images to inference. Specify if you
+                        want to use only part of numpy array
+  --batch_size BATCH_SIZE
+                        Number of images in a single request. Default: 1
+  --no-ssl              Set for non-SSL calls
+  --transpose_input     Set to make NCHW->NHWC input transposing
+  --no_imagenet         Set for non-Imagenet datasets
```
Comment (dtrawins) [inline examples/grpc_client/README.md]: This parameter name is confusing. More meaningful name would be show_imagenet_class or 'imagenet_classes'. The current behaviour is related to the model output  which includes imagenet classes.

**Pull request merged**